fast/stages-multitenant/0-bootstrap-tenant/variables.tf

```diff
@@ -266,7 +266,8 @@ variable "tenant_config" {
       gcp-network-admins  = optional(string)
       gcp-security-admins = optional(string)
     })
-    short_name = string
+    short_name           = string
+    short_name_is_prefix = optional(bool, false)
```

Comment:
I'm not seeing where the new attribute is used...

Reply:
Let me less brief: if we add this, I would assume we use the short name internally instead of the prefix. Which means anything in bootstrap that leverages prefix, and the actual prefix written to output files.

Reply:
Got it. As is the prefix will still be used even if that is set. So that would be to actually ignore the prefix and not just handle the case where prefix is null. Or we could just relax the validation right now to accept an empty prefix and don't introduce the boolean. That would at least match the implementation with the validation.

Reply:
Please take a look if that's what you meant.